app/javascript/components/reusable/withUserForm.js

```diff
@@ -878,8 +934,9 @@ const withUserForm = (WrappedComponent, schema, wrappedProps) => {
             errorText={ errors.programs }
             selectionRenderer={ this.selectionRenderer }
           >
-            { _.map(programs, ({ name, id }) => {
-              return <MenuItem key={ id } insetChildren checked={ _.indexOf(userPrograms, name) > -1 } value={ name } primaryText={ <span> { name } </span> } />;
+            { _.map(programs, ({ name, spanish_name, id }) => {
+              let programName = this.props.match.params[0] === ENGLISH ? name : spanish_name;
```

Comment:
Koy, considering that we intend to translate tutoria soon to other languages, we should start avoiding referring the language names inside the code. Do you think of any way it's possible doing this already here ?